web/src/upload.js

```diff
@@ -196,6 +196,8 @@ export async function upload_files(uppy, config, files) {
             break;
         }
     }
+    get_item("send_button", config).prop("disabled", true);
```

Comment:
Why did you move the disabling here? Is this functionally different than before starting the upload?

Reply:
As I mentioned in the PR description:
> The send button wasn't actually disabled when there is an upload in progress since the changes to the button would be undone by other input handlers.

I believe there is an input handler in `compose_validation` where it'll check if the character count in the compose box is over the limit or not. If it is, it'll disabled the button, if not it'll undo it. 

Previously, the button will be disabled before starting the upload of the files, which is great! But upon starting the upload there will be inputs into the compose box, then the input handler will pick it up and check. This results in the button being enabled again. This is counter intuitive and we should disable the button after the upload.

Reply:
Ah, I see what you're saying, thanks for clarifying further that it's the input from the uploading that does this.

We should handle this differently so that the button can still disable immediately. This PR still don't fix situations like the user typing new input while a file is uploading.

Reply:
Ah, that's a good point. I just expected the user to wait for the uploads to complete before typing. I guess a better way is to pass down something that'll help indicate if an file is being uploaded or not and then create a `should_disabled_button` function in `compose.js` that'll handle whether to disabled the button or not. Thoughts?